src/quo/components/list/item.cljs

```diff
@@ -86,7 +86,7 @@
          icon
 
          (keyword? icon)
-         (if new-ui?
+         (when new-ui?
```

Comment:
Also, when `new-ui?` is true, why empty space is shown instead of icon?

Reply:
> Hi, this is not the right way to fix this issue. `(if cond a b)` returns a when cond is true and b when false `(when cond a b)` `when` always returns last value when true and else nil. here a is component is for new-ui and b is component for old-ui. So yes, returning old-ui component will look like it fixes the issue. But it will break for new-ui. (We need to check why cond (new-ui?) is true in case of old-ui)



Makes sense, I will look into why `new-ui?` is `true`

Reply:
After more debugging I found out that the ` new-ui? ` code block in items component which was introduced  does nothing good either for new-ui or for old ui and hence I have removed it and icons work fine for both new/old ui.

@jo-mut  can you confirm please?

Reply:
removing the `new-ui?` would solve the regression but the same problem would not be on the redesign because icons  for the old component dont work with the redesign